### PR TITLE
Added neutron_ovs_agent_launcher.sh neutron_ovs_agent_start.sh

### PR DESCRIPTION
**Pull request by lennybe**
Initial commit to master

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    - DOCKER_VERSION=latest
+
+services:
+  - docker
+
+before_deploy:
+  - docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
+
+script:
+  - sudo docker pull arm64v8/centos:7
```
Comment (moshe010) [inline .travis.yml]: you are not using the Arch/Distro/Version variable here

Comment (moshe010) [reply .travis.yml]: it ok to start it just for centos, but let remove Arch/Distro/Version from the global

File: .travis.yml [hunk 2]
```diff
@@ -7,10 +7,8 @@ env:
   global:
     - ARCH=${TRAVIS_ARCH}
```
Comment (moshe010) [inline .travis.yml]: what about the ARCH is hardcoded in Line 21?

Comment (lennybe) [reply .travis.yml]: removed ARCH from globals

**Pull request merged**